app/src/main/res/values/strings.xml

```diff
@@ -302,10 +303,11 @@
     <string name="education_create_database_title">Create your database file</string>
     <string name="education_create_database_summary">You don\'t know KeePass DX yet, create your first password management file.</string>
     <string name="education_select_database_title">Open an existing database</string>
-    <string name="education_select_database_summary">Open your earlier KDBX file from your file browser to continue using your old database.</string>
+    <string name="education_select_database_summary">Open your earlier .kdbx or .kdb datafile from your file manager to continue using your old passwords.</string>
```

Comment:
Why old passwords, if they are old, we do not use them anymore. **Open your earlier .kdbx or .kdb datafile from your file manager to continue using your saved passwords.**